### PR TITLE
Update locale specific textures on locale change

### LINKED ISSUE #1099: Change language Doesn't Apply changed icon language
### Describe the problem
See title.
### Steps to reproduce
=== Default is SChinese ===

=== Change to English ===

### Notes or questions?
TMPE Version: 24a7b158832b2709d312606452bcde4938eeb6dc

### PR DESCRIPTION
**Pull request by krzychu124**
Resolves #1099 

I noticed some changes in #1052 so this fix might need a little adjusting before merging

### CODE REVIEW

**Review by kvakvs — APPROVED** [review]

File: TLM/TLM/UI/Textures/RoadUI.cs [hunk 1]
```diff
@@ -69,5 +69,20 @@ public static class RoadUI {
             VehicleInfoSignTextures[ExtVehicleType.Taxi] = LoadDllResource("RoadUI.taxi_infosign.png", signSize);
             VehicleInfoSignTextures[ExtVehicleType.Tram] = LoadDllResource("RoadUI.tram_infosign.png", signSize);
        }
+
+        public static void ReloadTexturesWithTranslation() {
+            IntVector2 size = new IntVector2(200);
```
Comment (DaEgi01) [inline TLM/TLM/UI/Textures/RoadUI.cs:74]: wtf is this? :D
... we should use something like ValueTuple<T1, T2> instead.
but ok .. not part of this PR.

**Review by DaEgi01 — APPROVED** [review]

File: TLM/TLM/UI/Textures/RoadUI.cs (hunk 1 above)
Comment (kvakvs) [reply TLM/TLM/UI/Textures/RoadUI.cs:74]: This is like Vector2 but for ints with exactly same meaning, it is x and y, or width and height together. Are tuples even supported in the C# version we're using? Why would I want a tuple or a new generic class name for something that only is used for width;height?

Comment (krzychu124) [reply TLM/TLM/UI/Textures/RoadUI.cs:74]: `ValueTuple<>` is .net 4.7+ so... nope :)

Comment (DaEgi01) [reply TLM/TLM/UI/Textures/RoadUI.cs:74]: yeah it is .. but its easy to implement our own.

Comment (DaEgi01) [reply TLM/TLM/UI/Textures/RoadUI.cs:74]: @kvakvs tuples are just generic structs. sure they are supported by c#.

Comment (kvakvs) [reply TLM/TLM/UI/Textures/RoadUI.cs:74]: Then you can assume i implemented our own. It carries meaning of width;height pair

Comment (DaEgi01) [reply TLM/TLM/UI/Textures/RoadUI.cs:74]: https://github.com/CitiesSkylinesMods/TMPE/pull/1122
